Article/Article.js

```diff
@@ -26,4 +33,8 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");
+
+articles.forEach( (domElement) => {
```

Comment:
Can be simplified to: `articles.forEach(domElement  => new Article(domElement ))`